lib/collections.js

```diff
@@ -17,6 +17,18 @@ var normalizedWords = [];
 
 /* your code starts here */
 
+// split the paragraphs
+var words = paragraph.split(/\s+/);
+
+// getting rid of punctuation
+for (var i = 0; i < words.length; i++) {
+  normalizedWords[i] = words[i].replace(/[^\w']+/, '');
+}
+
+// make everything lower case
+for (var i = 0; i < normalizedWords.length; i++) {
+  normalizedWords[i] = normalizedWords[i].toLowerCase();
+}
```

Comment:
Yep, this will work! In fact, we could have `replace`d and `toLowerCase`d in a single step, if we wanted, by chaining those two operations:

`words[i].replace(/[^\w']+/, '').toLowerCase();`